rails-time-track/app/controllers/api/users_controller.rb

```diff
@@ -1,13 +1,50 @@
 module Api
   class Api::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :update, :destroy]
 
     def index
       render json: User.all
     end
 
+    def create
+      user = User.new(user_params)
+      if user.save
+        render json: user, status: :created
+      else
+        render json: { errors: user.errors}, status: :unprocessable_entity
```

Comment:
Hi, I comment that there is a method named "render_errors" which render errors with a specific structure and with json format, you could use that to have a homogeneous code to show errors.